code/pyproject.toml

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+[build-system]
+requires = ["setuptools"]
```

Comment:
I see that the tutorial puts a lower-bound limit on the dependency
```toml
requires = ["setuptools>=61.0"]
```
can't we do the same?